accounts/manager.go

```diff
@@ -63,7 +54,7 @@ type Manager struct {
 
 // NewManager creates a generic account manager to sign transaction via various
 // supported backends.
-func NewManager(config *Config, backends ...Backend) *Manager {
+func NewManager(backends ...Backend) *Manager {
```

Comment:
I hate to be that guy again, but this function has had a stable signature for a very long time. So I would prefer not to change it. We can simply ignore the setting.

Reply:
It's a bit weird that we even added it. In hindsight it would've been better to add a method like `SetInsecureUnlockAllowed` on the `Manager`.

Reply:
I guess not, for the same reason. Ugh